### PR TITLE
Closes #1648 - Environment Update Instructions

### LINKED ISSUE #1648: Environment Update Instructions
The documentation only provides instructions on install arkouda and configuring the environment initially. The commands to update the environment in the event of dependency updates are not provided. These instructions are the same for each OS, but differ depending upon if `conda` or `pip` is being used primarily for environment management.

Environment update instructions need to be added to `INSTALL.md`

### PR DESCRIPTION
**Pull request by Ethan-DeBandi99**
Closes #1648 

- Adds environment update instructions for `conda` 
- Adds environment update instructions for `pip`

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -333,6 +336,32 @@ pip install -e .
 pip install -e .[dev] 
 ```
 
+<a id="env-upd"></a>
+## Updating Environment <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+As Arkouda progresses through its life-cycle, dependencies may change. As a result, it is recommended that you keep your development environment in sync with the latest dependencies. The instructions vary depending upon you preffered environment management tool.
```
Comment (pierce314159) [inline INSTALL.md:341]: typo in `preffered` -> `preferred`

File: INSTALL.md [hunk 2]
```diff
@@ -333,6 +336,32 @@ pip install -e .
 pip install -e .[dev] 
 ```
 
+<a id="env-upd"></a>
+## Updating Environment <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+As Arkouda progresses through its life-cycle, dependencies may change. As a result, it is recommended that you keep your development environment in sync with the latest dependencies. The instructions vary depending upon you preffered environment management tool. 
+
+<a id="env-upd-conda"></a>
+### Conda <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+*If you provided a different name when creating the environment, place `arkouda-dev` or `arkouda` with the name of your Conda environment.*
```
Comment (pierce314159) [inline INSTALL.md:345]: typo in `place` -> `replace`

**Review by pierce314159 — COMMENTED** [review]
2 small typos. Looks good otherwise!

File: INSTALL.md (hunk 1 above)
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Fixed

File: INSTALL.md (hunk 2 above)
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Fixed

**Review by pierce314159 — APPROVED** [review]

**Review by joshmarshall1 — APPROVED** [review]
Looks good

**Pull request merged**